### ISSUE TITLE
Extend status command to provide details about the VM

### ISSUE THREAD

**Issue by gbraad** [minishift/minishift#1247]
We should extend the `minishift status` command to show some basic information about the VM, besides the state. Some of this could be the IP address, to align with `minikube status`, but also `minishift status --check` could be introduced to do some of the `preflightChecksAfterStartingHost`, like IP reachability check, disk usage, or provide information about the RHSM subscription status for RHEL images.

**Comment by gbraad** [comment]
Ref: https://github.com/kubernetes/minikube/blob/master/cmd/minikube/cmd/status.go#L81

**Comment by hferentschik** [comment]
Duplicate of #373

**Comment by hferentschik** [comment]
I'd also like to see a differentiation between VM and OpenShift state:

    $ minishift status
    VM: Running 
    OpenShift: [Running|Stopped|Unavailable]

**Comment by gbraad** [comment]
+1 Good point

Can you explain the status Unavailable?

**Comment by hferentschik** [comment]
I meant something like "Error" or "Unkown". But this was just a example. On second thought "Unavailable" could also mean "Not deployed". The VM is running, but no Origin container around.

**Comment by gbraad** [comment]
Exactly, that is what I thought. There are several states possible and
definitely would like to explore them more.

Is there a way to extract the version info from the origin/OCP container?

**Comment by hferentschik** [comment]
> Exactly, that is what I thought. There are several states possible and definitely would like to explore them more.

Right, the same way that there are several other states than 'Running' and 'Stopped' for the libmachine VM. We just tend to ignore them. 

> Is there a way to extract the version info from the origin/OCP container?

Yes, we already do that

**Comment by gbraad** [comment]
> Yes, we already do that

haven‘t noticed.

**Comment by hferentschik** [comment]
https://github.com/minishift/minishift/blob/master/cmd/minishift/cmd/openshift/version.go#L50

**Comment by gbraad** [comment]
Ah, just a simple:

```
$ docker exec origin openshift version
```

We definitely need to use this...

**Comment by gbraad** [comment]
```
$ minishift status
VM: Running
OpenShift: Running... v3.6.1
Storage: 10% of 20GB
```

**Comment by LalatenduMohanty** [comment]
Moving to the next milestone because I think we should do it after #126 is fixed.

**Comment by praveenkumar** [comment]
Resolved by #1425
